### DISCUSSION TITLE
Allow Enterprise to use minor updates

### DISCUSSION THREAD

**Discussion by Nisgrak** [dbeaver/dbeaver#11291] [Ideas]
**Is your feature request related to a problem? Please describe.**
The Community edition(7.3.4) has minor updates with QoL changes much more frecuent than Enterprise edition(7.3.0).

**Describe the solution you'd like**
No need to wait 3 months between minor version.

**Describe alternatives you've considered**
Use the Early Access, but I want tested changes not experimental ones, only follow the Community edition improvements.

**Comment by kseniiaguzeeva** [comment]
Thank you for your suggestion. And you already wrote the answer-  please use Early Access. The optimal way is download it right after CE releases (this version should be more stable). We do not plan to release minor versions of Enterprise Edition yet.
